### DISCUSSION TITLE
Moving from local install to Docker

### DISCUSSION THREAD

**Discussion by commputethis** [josephdadams/TallyArbiter#462] [General]
I am new to Docker and thought others might benefit from some additional information that would have helped me to more easily migrate from TallyArbiter locally installed on an RPi to a Docker container running on an RPi.  I had to copy my config.json file from ~/.local/share/TallyArbiter/ to ~/TallyArbiter for the container to get my previous settings.

Also, I modified the command to run the container to the following so it is easier to stop the container if/when needed by including the `--name` command.

`docker run --name tally_arbiter -d -p 4455:4455 -v $(pwd):/app/config --env APPDATA=/app/config --restart unless-stopped josephdadams/tallyarbiter`

Hope this helps others save a little time and frustration.

**Comment by JTF4** [comment]
Great guide to simplify migrating from local to docker! Thanks for your contribution!
